### ISSUE TITLE
sitewide, forms — 508-defect-3 [SCREENREADER]: Decorative content MUST be implemented in a way it can be ignored by assistive technology

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#9113]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

> Found during [a11y spot check for VSA BAM2 MDOT MVP](https://github.com/department-of-veterans-affairs/va.gov-team/issues/5868)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

On the buttons there is a small arrow icon.  These icons should be hidden from screen readers, as they are decorative. This guidance applies to any instance of the arrow icon on the buttons throughout the product.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to  read the content of the page without extraneous noise from decorative images.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all
* Server destination: all

## Steps to Recreate

1. Enter https://staging.va.gov/hearing-aid-batteries-and-accessories/introduction in browser, 
2. Log in as username: vets.gov.user+1@gmail.com
3. Start screen reading device
4. Read content of the page
5. Verify that the arrows are read by the screen reader

## Possible Fixes (optional)

**Current code**
```html
<span class="button-icon"> »</span>
```

**Recommended code**
```html
<span aria-hidden="true" role="presentation" class="button-icon"> »</span>
```

## WCAG or Vendor Guidance (optional)

* [W3C WCAG 1.1.1 - Failure of Success Criterion 1.1.1 due to not marking up decorative images in HTML in a way that allows assistive technology to ignore them](https://www.w3.org/TR/WCAG20-TECHS/F38.html)

* [W3C Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)

* [HTML5 Accessibility: aria-hidden and role=”presentation”](http://john.foliot.ca/aria-hidden/)

## Screenshots or Trace Logs

![Screen Shot 2020-05-12 at 11 30 48 AM](https://user-images.githubusercontent.com/57469/81713573-0e7d4880-9444-11ea-9200-711eaa56164c.png)

**Comment by humancompanion-usds** [comment]
We have web components for buttons and links now. There are no icons in buttons currently. We do have links that include icons and those do hide the icons from screen readers. Closing.
